### PR TITLE
Dependencies update

### PR DESCRIPTION
**Pull request by MaxFrax**
Updates all the dependencies to the latest version possible.
I have run all the pipeline with success with the dependencies updated.
Closes also #380

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,93 +1,105 @@
...
 sphinxcontrib-qthelp==1.0.2
 sphinxcontrib-serializinghtml==1.1.3
-SQLAlchemy==1.3.5
-tensorboard==1.13.1
-tensorflow==1.13.1
-tensorflow-estimator==1.13.0
+SQLAlchemy==1.3.13
+tensorboard==2.1.0
+tensorflow==2.1.0
+tensorflow-estimator==2.1.0
```
[the remark this answers is not in the document]
Comment (marfox) [reply requirements.txt]: I'll also dig deeper into the CLI commands that trigger machine learning stuff.

File: soweego/wikidata/api_requests.py [hunk 2]
```diff
@@ -357,6 +357,9 @@ def _lookup_label(item_value):
     if not response_body:
         LOGGER.warning('Failed label lookup for %s', item_value)
         return None
+    if not response_body.get('entities'):
+        LOGGER.warning('No entities for %s', item_value)
+        return None
```
Comment (marfox) [inline soweego/wikidata/api_requests.py]: We should actually refactor this, because it looks like duplicate code.
Why don't we reuse `_sanity_check`?

Comment (MaxFrax) [reply soweego/wikidata/api_requests.py]: I'll look into it, but you know the validator way better than me.

Comment (marfox) [reply soweego/wikidata/api_requests.py]: Don't worry, I'm gonna fix this.

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (MaxFrax) [reply requirements.txt]: My test has been running the whole pipeline we usually run in production.
In that scenario, everything worked.
If you can check the machine learning CLI commands, that would be awesome.
What kind of issues did you have? Was it just functions signatures or something about the correctness of results?

[the remark this answers is not in the document]
Comment (marfox) [reply requirements.txt]: I don't remember, so I'll try all CLI commands related to machine learning

**Comment by marfox** [comment]
I'm on the prod VPS. If I run:
```
$ ./docker/prod.sh
root@ff0fb7a4a2db:/app/soweego# python -m soweego
```
I get unrelevant tensorflow warnings:
```
2020-03-18 18:48:04.450180: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer.so.6'; dlerror: libnvinfer.so.6: cannot open shared object file:
 No such file or directory                                    
2020-03-18 18:48:04.452246: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer_plugin.so.6'; dlerror: libnvinfer_plugin.so.6: cannot open share
d object file: No such file or directory     
2020-03-18 18:48:04.452284: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:30] Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
```
We should definitely silence them.

File: soweego/cli.py [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
 # set env variable to ignore tensorflow warnings
 # (only errors are printed)
-tf.logging.set_verbosity(tf.logging.ERROR)
+tf.get_logger().setLevel(logging.ERROR)
```
Comment (marfox) [inline soweego/cli.py]: This seems to be not working, see https://github.com/Wikidata/soweego/pull/384#issuecomment-600806421

**Comment by marfox** [comment]
We should also keep an eye on https://www.tensorflow.org/guide/migrate

**Comment by marfox** [comment]
**IMPORTANT:** we gave up switching from `keras` to `tensorflow.keras`, since we're hitting an open tensorflow issue, see efd8f62ad9ff5432ae66a514c27c5cc6fcdc7433

This PR introduces the following modifications:
- refactor WD API sanity checks: 287f8ba38b3d7a48c73aefa7c682127ed22d8cd3
- silence tensorflow and mlens: b4ecdc49375f5cce58e24e7d6fb769a3899db3a9 and a747d6a608a0e25de1c765e7151c2fe300fa75e2
- tell `isort` not to sort imports in `soweego/cli.py`, because silence must be before import: 3556fff7a1f36ad1ea547ea7869161fe6e6563e2
- turn on `keras` printing when training. More output, but more control: fbb78e2f4e7b703e53e7c572bac7388a28d3c78f
- let `pip-tools` properly handle dependency pinned versions: 26de3142c888b4afd743d7d0514fabd44b9bb4f2
- style enforcing commands script: 23afc3ef5399eae5a4c5f25081424731d04e931c

**Pull request merged**